cmd/kubeadm/app/features/features.go

```diff
@@ -31,18 +31,14 @@ const (
 	// CoreDNS is GA in v1.11
 	CoreDNS = "CoreDNS"
 
-	// DynamicKubeletConfig is beta in v1.11
-	DynamicKubeletConfig = "DynamicKubeletConfig"
-
 	// Auditing is beta in 1.8
 	Auditing = "Auditing"
 )
 
 // InitFeatureGates are the default feature gates for the init command
 var InitFeatureGates = FeatureList{
-	CoreDNS:              {FeatureSpec: utilfeature.FeatureSpec{Default: true, PreRelease: utilfeature.GA}},
-	DynamicKubeletConfig: {FeatureSpec: utilfeature.FeatureSpec{Default: false, PreRelease: utilfeature.Beta}},
```

Comment:
sorry, just realized that we cannot remove it directly given the gate is BETA already.

we have to fist deprecate it and keep it around for 2 releases given the original feature still exists in the kubelet:
https://github.com/kubernetes/website/blob/master/content/en/docs/reference/using-api/deprecation-policy.md#deprecation

i tried to explain that in the last two meetings.

/hold
/assign @timothysc

Reply:
What we will do? Reopen the phase pr and mark the feature gate as deprecated?

Reply:
i will leave to the others for further comments.
@luxas @fabriziopandini